ocpmodels/models/dimenet.py

```diff
@@ -96,7 +96,7 @@ def forward(self, data):
             self.interaction_blocks, self.output_blocks[1:]
         ):
             x = interaction_block(x, rbf, sbf, idx_kj, idx_ji)
-            P += output_block(x, rbf, i)
+            P += output_block(x, rbf, i, num_nodes=data.natoms.sum())
```

Comment:
@mshuaibii was this the only fix for the DimeNet dimensionality issue you were running into?

I faced the same issue for my IS2RE models and this fixes it.

Reply:
Yup exactly - to be consistent with line 92 I was using `pos.size(0)`